server/server.js

```diff
@@ -12,16 +12,17 @@ app.use((_, res, next) => {
 });
 
 app.use(logger('dev'));
-app.use(express.json({urlencoded:true}))
+app.use(express.json({ urlencoded: true }))
 // You can place your routes here, feel free to refactor:
-const { example, seedTopics, search, addFavorite, deleteFavorite, getAllTopics, news } = require('./routes');
+const { example, seedTopics, search, addFavorite, deleteFavorite, getAllTopics, news, charity } = require('./routes');
 app.use('/api/example', example)
 app.use('/api/seedTopics', seedTopics)
 app.use('/api/searchTopics', search)
 app.use('/api/addFavorites', addFavorite)
 app.use('/api/deleteFavorites', deleteFavorite)
 app.use('/api/getAllTopics', getAllTopics);
 app.use('/api/news', news);
+app.use(`/api/charity`, charity)
```

Comment:
Change this to use RESTful naming convention. Other than that, everything else is good to go. Nice test!